a/asusd/stone.yaml

```diff
@@ -0,0 +1,41 @@
+#
+# SPDX-FileCopyrightText: © 2020-2024 Serpent OS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+name        : asusd
+version     : 6.1.0-rc2
+release     : 1
+summary     : Control ASUS laptop features easily
+license     :
+    - MPL-2.0
+homepage    : https://asus-linux.org/
+upstreams   :
+    - git|https://gitlab.com/asus-linux/asusctl.git : d785e17f9518c414134ce00585f1d83c0698090c
+description : |
+    Control ASUS platform features such as LEDs, AniMe, fans and more.
+builddeps   :
+    - binary(cargo)
+    - binary(cc)
+    - binary(clang)
+    - binary(make)
```

Comment:
These four dependencies can most likely be removed. 

- `clang` is pulled in automatically as a dependency of cargo/rust. 
- `cc` shouldn't be required if the makefiles/scripts are respecting the value of the CC and CXX environmental variables (and if they're not then most likely they're not respecting other variables like CFLAGS/CXXFLAGS/LDFLAGS and should be fixed). 
- Boulder will automatically add builddeps when certain macros are used. `%cargo_fetch` automatically adds a dependency on cargo, `%make` and `%make_install` add make.